docker/prod/nodejs_v2/Dockerfile

```diff
@@ -7,7 +7,7 @@ COPY frontend_v2/package.json frontend_v2/yarn.lock /code/
 RUN yarn install
 ENV PATH="/code/node_modules/.bin:$PATH" 
 COPY frontend_v2/ /code
-RUN ng build --prod
+RUN ng build --prod --configuration=${NODE_ENV}
```

Comment:
Can we please define this environment variable first before using it?

Reply:
It's already there https://github.com/Cloud-CV/EvalAI/blob/master/docker-compose-staging.yml#L69

Reply:
No, you will have to add it like this -- https://github.com/Cloud-CV/EvalAI/blob/c3898a570c5b515a2a2f2f80aa2fa2fec61a9cd0/docker/prod/nodejs/Dockerfile#L3